content/docs/connect/connect-from-any-app.md

```diff
@@ -54,6 +54,9 @@ psql postgres://daniel:<password>@ep-restless-rice.us-east-2.aws.neon.tech/neond
 ```
 
 <Admonition type="note">
+Neon requires that all connections use SSL/TLS encryption, but you can increase the level of protection by appending an `sslmode` parameter setting to your connection string. For instructions, see [Connect to Neon securely](/docs/connect/connect-securely).
+
+Please be aware that some older client libraries and drivers, including older `psql` executables, are built without SNI support and require a workaround. For more information, see [Connect from older clients](/docs/connect/connectivity-issues).
```

Comment:
Why have you added this passage? For me, it says pretty much the same as the next one

Reply:
Hi Vadim,
My intention here is to let users know that they can add an sslmode parameter to their connection string to increase the security of their SSL/TLS connection and direct them to the instructions from other topics that deal with connections.

As I understand it, SNI (if that's what you are referring to) is just a means of routing connections to the correct compute endpoint. SNI is an extension of TLS, but has no bearing on the level of security of your SSL/TLS connection. But please correct me if I am wrong!

On a related note, there was some discussion in the past as to whether we should include sslmode in the connect string examples in the docs and elsewhere. The added passage is intended to address this issue to a degree.

Reply:
try to read this note altogether:

```
Neon requires that all connections use SSL/TLS encryption, but you can increase the level of protection by appending an `sslmode` parameter setting to your connection string. For instructions, see [Connect to Neon securely](/docs/connect/connect-securely).

Please be aware that some older client libraries and drivers, including older `psql` executables, are built without SNI support and require a workaround. For more information, see [Connect from older clients](/docs/connect/connectivity-issues).

For information about connecting with `psql`, see [Connect with psql](/docs/connect/query-with-psql-editor). Please be aware that some older client libraries and drivers, including older `psql` executables, are built without [Server Name Indication (SNI)](/docs/reference/glossary#sni) support and require a workaround. For more information, see [Connect from older clients](/docs/connect/connectivity-issues).
```

For me, it explains the same things and refers to the same docs. it was hard to understand the final idea behind this

Reply:
Thank you. I see that some text was repeated there, and this note was getting a bit too large. 
- Removed the repeated text
- Left the note about using sslmode 
- Moved other items to a Connection notes section toward the bottom.